.github/workflows/ci.yml

```diff
@@ -4,25 +4,27 @@ on: [push, pull_request]
 
 jobs:
   ci:
-    runs-on: ubuntu-latest
+    name: Erlang ${{matrix.otp-version}} / Elixir ${{matrix.elixir-version}}
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        elixir-version: [1.12.x, 1.11.x, 1.10.x, 1.9.x, 1.8.x]
+        # https://hexdocs.pm/elixir/compatibility-and-deprecations.html
+        elixir-version: [1.17.x, 1.16.x, 1.15.x, 1.14.x, 1.13.x]
         include:
-          - elixir-version: 1.12.x
+          - elixir-version: 1.17.x
```

Comment:
We should also include OTP 26 and 27 as well?

Reply:
Yep absolutely, I just wanted to make all OTP 25 builds go green and then check for 26 and 27 after 🙂

Reply:
@kianmeng I added a few!

```yaml
- elixir-version: 1.17.x
  otp-version: 27.x
- elixir-version: 1.17.x
  otp-version: 26.x
- elixir-version: 1.16.x
  otp-version: 26.x
- elixir-version: 1.15.x
  otp-version: 26.x
```

🙌